src/test/java/uk/gov/hmcts/juror/api/moj/report/grouped/UnpaidAttendanceSummaryReportTest.java

```diff
@@ -0,0 +1,170 @@
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupedTableData;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.domain.QAppearance;
+import uk.gov.hmcts.juror.api.moj.domain.QJuror;
+import uk.gov.hmcts.juror.api.moj.enumeration.AppearanceStage;
+import uk.gov.hmcts.juror.api.moj.report.AbstractGroupedReportTestSupport;
+import uk.gov.hmcts.juror.api.moj.report.DataType;
+import uk.gov.hmcts.juror.api.moj.report.ReportGroupBy;
+import uk.gov.hmcts.juror.api.moj.repository.PoolRequestRepository;
+import uk.gov.hmcts.juror.api.moj.service.CourtLocationService;
+import uk.gov.hmcts.juror.api.moj.utils.SecurityUtil;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@SuppressWarnings({
+    "PMD.LawOfDemeter",
+    "PMD.ExcessiveImports"
+})
+class UnpaidAttendanceSummaryReportTest extends AbstractGroupedReportTestSupport<UnpaidAttendanceSummaryReport> {
+
+    private MockedStatic<SecurityUtil> securityUtilMockedStatic;
+    private CourtLocationService courtLocationService;
+
+    public UnpaidAttendanceSummaryReportTest() {
+        super(QAppearance.appearance,
+            UnpaidAttendanceSummaryReport.RequestValidator.class,
+            ReportGroupBy.builder()
+                .dataType(DataType.POOL_NUMBER_BY_APPEARANCE)
+                .removeGroupByFromResponse(true)
+                .build(),
+            DataType.JUROR_NUMBER,
+            DataType.FIRST_NAME,
+            DataType.LAST_NAME);
+    }
+
+    @BeforeEach
+    @Override
+    public void beforeEach() {
+        this.securityUtilMockedStatic = mockStatic(SecurityUtil.class);
+        this.courtLocationService = mock(CourtLocationService.class);
+        super.beforeEach();
+    }
+
+    @AfterEach
+    void afterEach() {
+        securityUtilMockedStatic.close();
+    }
+
+    @Override
+    public UnpaidAttendanceSummaryReport createReport(PoolRequestRepository poolRequestRepository) {
+        return new UnpaidAttendanceSummaryReport(poolRequestRepository, this.courtLocationService);
+    }
+
+    @Override
+    protected StandardReportRequest getValidRequest() {
+        return StandardReportRequest.builder()
+            .reportType(report.getName())
+            .fromDate(LocalDate.of(2020, 1, 1))
+            .toDate(LocalDate.of(2020, 1, 2))
+            .build();
+    }
+
+    @Override
+    @DisplayName("positivePreProcessQueryTypicalCourt")
+    public void positivePreProcessQueryTypical(JPAQuery<Tuple> query, StandardReportRequest request) {
+        request.setToDate(LocalDate.of(2023, 1, 1));
+        request.setToDate(LocalDate.of(2023, 1, 2));
```

Comment:
Should not need to add these here as they will default to what you specified in getValidRequest()

Reply:
Removed